influxdb.html

```diff
@@ -323,32 +482,39 @@
         <label for="node-input-name"><i class="fa fa-tag"></i> <span data-i18n="node-red:common.label.name"></span></label>
         <input type="text" id="node-input-name" data-i18n="[placeholder]node-red:common.label.name">
     </div>
-    <div class="form-tips" id="node-warning" style="display: none"><span data-i18n="[html]influxdb.querytip"></span></div>
+    <div class="form-tips" id="node-warning" style="display: none"><span data-i18n="[html]influxdb.tip.querytip"></span></div>
 </script>
 
 <script type="text/x-red" data-help-name="influxdb in">
+    <p><b>InfluxDB 1.8</b></p>
     <p>Allows basic queries to be made to an influxdb time series database.</p>
     <p>The query can be specified in the configuration property or using the property
-    <b>msg.query</b>.  The results will be returned in <b>msg.payload</b>.</p>
+    <b>msg.query</b>. The results will be returned in <b>msg.payload</b>.</p>
     <p>To output the results of the query in the raw output format returned by InfluxDb,
     check the <i>Raw Output</i> checkbox.</p>
     <p>Check <i>Advanced Query Options</i> to specify a time precision and retention policy for the query.<p>
-    <p>The raw output configuration can be overwritten by the message property <b>msg.rawOutput</b>. </p>
+    <p>The raw output configuration can be overwritten by the message property <b>msg.rawOutput</b>.</p>
     <p>The advanced query options <i>Time Precision</i> and <i>Retention Policy</i> can be overwritten using 
-    message properties <b>msg.precision</b> and <b>msg.retentionPolicy</b>.</p>  
+    message properties <b>msg.precision</b> and <b>msg.retentionPolicy</b>.</p>
+
+    <p>&nbsp;</p><p><b>InfluxDB 1.8 Flux and InfluxDB 2.0</b></p>
+    <p>Allows basic queries to be made to an influxdb 2 time series database.</p>
+    <p>The flux query can be specified in the configuration property or using the property
+    <b>msg.query</b>. The results will be returned in <b>msg.payload</b>.</p>
 </script>
 
 <script type="text/javascript">
     RED.nodes.registerType('influxdb in', {
-        category: 'storage-input',
+        category: 'storage-output',
```

Comment:
why is the input node in the 'storage-output' category

Reply:
Must be a typo